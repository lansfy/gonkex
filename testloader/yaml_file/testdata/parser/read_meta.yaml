- name: loader MUST read tests with metadata
  method: POST
  path: /some/path
  request: "requestbody"
  response:
    200: "aaabbb"
  meta:
    key1: value1
    key2: 44444
    key3:
      subkey1: subvalue1
      subkey2: subvalue2
    key4: [arr1, arr2]
