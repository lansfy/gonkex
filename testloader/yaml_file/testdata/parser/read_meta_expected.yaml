- GetName: loader MUST read tests with metadata
  GetMethod: POST
  Path: /some/path
  GetRequest: requestbody
  GetResponses:
    200: "aaabbb"
  GetMeta:
    key1: value1
    key2: 44444
    key3:
      subkey1: subvalue1
      subkey2: subvalue2
    key4: [arr1, arr2]
  GetFileName: testdata/parser/read_meta.yaml
  FirstTestInFile: true
  LastTestInFile: true
